strr-strata-web/app/pages/strata-hotel/dashboard/[registrationId].vue

```diff
@@ -63,13 +68,13 @@ useHead({
 
 definePageMeta({
   layout: 'connect-dashboard',
-  middleware: ['auth'],
-  path: '/strata-hotel/dashboard/:registrationId'
```

Comment:
is the 'path' not necessary due to the page naming convention?

Reply:
Yeah the path will be created using the file structure, or you can override it with the definePageMeta 'path' property